src/src/common.cpp

```diff
@@ -30,22 +32,28 @@ expresslrs_rf_pref_params_s ExpressLRS_AirRateRFperf[RATE_MAX] = {
 SX1280Driver DMA_ATTR Radio;
 
 expresslrs_mod_settings_s ExpressLRS_AirRateConfig[RATE_MAX] = {
-    {0, RADIO_TYPE_SX128x_FLRC, RATE_FLRC_1000HZ, SX1280_FLRC_BR_0_650_BW_0_6, SX1280_FLRC_BT_1, SX1280_FLRC_CR_1_2,     1000, TLM_RATIO_1_128,  2, 32, 8, 1},
-    {1, RADIO_TYPE_SX128x_FLRC, RATE_DVDA_500HZ,  SX1280_FLRC_BR_0_650_BW_0_6, SX1280_FLRC_BT_1, SX1280_FLRC_CR_1_2,     1000, TLM_RATIO_1_128,  2, 32, 8, 2},
-    {2, RADIO_TYPE_SX128x_FLRC, RATE_DVDA_250HZ,  SX1280_FLRC_BR_0_650_BW_0_6, SX1280_FLRC_BT_1, SX1280_FLRC_CR_1_2,     1000, TLM_RATIO_1_128,  2, 32, 8, 4},
-    {3, RADIO_TYPE_SX128x_LORA, RATE_LORA_500HZ,  SX1280_LORA_BW_0800,         SX1280_LORA_SF5,  SX1280_LORA_CR_LI_4_6,  2000, TLM_RATIO_1_128,  4, 12, 8, 1},
-    {4, RADIO_TYPE_SX128x_LORA, RATE_LORA_250HZ,  SX1280_LORA_BW_0800,         SX1280_LORA_SF6,  SX1280_LORA_CR_LI_4_7,  4000, TLM_RATIO_1_64,   4, 14, 8, 1},
-    {5, RADIO_TYPE_SX128x_LORA, RATE_LORA_150HZ,  SX1280_LORA_BW_0800,         SX1280_LORA_SF7,  SX1280_LORA_CR_LI_4_7,  6666, TLM_RATIO_1_32,   4, 12, 8, 1},
-    {6, RADIO_TYPE_SX128x_LORA, RATE_LORA_50HZ,   SX1280_LORA_BW_0800,         SX1280_LORA_SF9,  SX1280_LORA_CR_LI_4_6, 20000, TLM_RATIO_NO_TLM, 2, 12, 8, 1}};
+    {0, RADIO_TYPE_SX128x_FLRC, RATE_FLRC_1000HZ,    SX1280_FLRC_BR_0_650_BW_0_6, SX1280_FLRC_BT_1, SX1280_FLRC_CR_1_2,    TLM_RATIO_1_128,  8,  1000, 32, OTA4_PACKET_SIZE, 1},
+    {1, RADIO_TYPE_SX128x_FLRC, RATE_FLRC_500HZ,     SX1280_FLRC_BR_0_650_BW_0_6, SX1280_FLRC_BT_1, SX1280_FLRC_CR_1_2,    TLM_RATIO_1_128,  4,  2000, 32, OTA4_PACKET_SIZE, 1},
```

Comment:
As part of the DVDA PR I changed the F1000 FHSShopInterval to 2.  When I saw dropped backs there was generally a chunk lost, most likely due to interference on that freq.  Lowering FHSShopInterval reduced the number of lost packets.  I believe JamesK also observed the same thing.

Reply:
Merge conflict mistake on my part. Should F500 keep 4 hops like it had before it was removed, or be dropped to 2? The duty cycle on F500 is less than 20% which may be more immune to that interference. I'll make it whatever you feel is best though.

Reply:
I would stick with 2 for consistency.